docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+    ttyS0::respawn:/sbin/getty -L ttyS0 115200 vt100
+    
+    # Stuff to do for the 3-finger salute
+    ::ctrlaltdel:/sbin/reboot
+    
+    # Stuff to do before rebooting
+    ::shutdown:/sbin/rc shutdown
+    ~~~
+
+3.  Create the Grub 2 boot config
+        mkdir /alpine/boot/grub
+{: .file }
+/alpine/boot/grub/grub.cfg
+:   ~~~ conf
+    set root=(hd0)
+    set default="Alpine Linux"
+    set timeout=0
+    
+    menuentry "Alpine Linux" {
+              linux /vmlinuz-grsec root=/dev/sdb modules=sd-mod,usb-storage,ext4 console=ttyS0 quiet
+              initrd /initramfs-grsec
+    }
+    ~~~
+    
+4.  Configure mkinitfs
+        mkdir /alpine/etc/mkinitfs
+{: .file }
+/alpine/etc/mkinitfs/mkinitfs.conf
+:   ~~~ conf
+    features="ata ide scsi virtio base ext4"
+    ~~~
+
+5.  Copy resolv.conf in
+        cp /etc/resolv.conf /alpine/etc
+        
+6.  Add `ttyS0` to securetty to allow root login over Lish
+        echo ttyS0 >> /alpine/etc/securetty
+
+### Installing the kernel and creating the initramfs
+
+1.  Bind proc and dev into the chroot
+        mount --bind /proc /alpine/proc
+        mount --bind /dev /alpine/dev
+
+2.  Enter the chroot
+        chroot /alpine /bin/sh
+
+3.  Select a mirror.  You can use the `f` option to automatically pick the fastest mirror.
+        setup-apkrepos
+
+4.  Update packages
+        apk update
+
+5.  Setup your hostname.
+        setup-hostname -n `hostname`
+
+6.  Configure important services to start automatically.
+        rc-update add networking boot
+        rc-update add urandom boot
+        rc-update add cron
+
+7.  Install the kernel
+        apk add linux-grsec
+
+8.  Exit and shutdown
+        exit
+        shutdown -h now
+        
+## Final Configuration
+
+### Reboot into alpine
+
+1.  Reboot into the Alpine Linux configuration you made earlier.  If it's the only one, this can be accomplished from Lish via `boot 1`
+
+2.  Login as root.  By default there is no password for root.
+
+### Configuration
+
+1.  Set a password for root
+        passwd
+
+2.  Create a user
+        adduser `username`
+
+3.  Install sudo
+        apk add sudo
+
+4.  Configure sudo to allow users in the group sudo to use sudo.  
+        echo "%sudo   ALL=(ALL) ALL" >> /etc/sudoers
+    Alternatively, you can allow passwordless sudo
+        echo "%sudo   ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+5.  Create the sudo group and add your user to it.
+        addgroup sudo
+        adduser `username` sudo
+        
+6.  Setup sshd.  Alpine has a simple setup script to handle this.  I recommend the `openssh` server if you want sftp access, though `dropbear` is lighter if you just need ssh access
+        setup-sshd
+
+## Finished
+
+### Next Steps
+
+At this point, you should be able to ssh into your server.  Alpine is very lightweight, and doesn't install very much unless you ask it to.
+For instance, you won't have a text editor to start off, and the only service running is your sshd.
```

Comment:
It does seem like a brief explanation of the package manager, if any, would be expected here.